File: hw1/c++/hw1.cpp
<<<<<<< HEAD
#include <iostream>
#include <fstream>

using namespace std;

int main() 
{
	
	int data[50];
	int buffer;
	int count = 0;

	ifstream fcin("input.txt");

	while (fcin >> buffer)
	{
		//cout << buffer << endl;
		data[count] = buffer;
		count++;
	}
	
	int min = data[0];
	int max = data[1];
	int ans = data[2];
	int input;
	
	cout << "Please guess an integer number between " << min << " and " << max << endl; 
	cin >> input;
	while(input != ans){
		if(input < min || input > max){
			cout << "The number you entered is not within the valid range." << endl;
			cout << "Please try again!" << endl;
		}
		else if(input < ans){
			min = input;
		}
		else if(input > ans){
			max = input;
		}
		cout << "Please guess an integer number between " << min << " and " << max << endl; 
		cin >> input;
	}
	cout << "Bingo! You hit the bomb." << endl; 

	system("pause");
	return 0;
}

=======
#include <iostream>
#include <fstream>

using namespace std;

int main() 
{
	
	int data[50];
	int buffer;
	int count = 0;

	ifstream fcin("input.txt");

	while (fcin >> buffer)
	{
		//cout << buffer << endl;
		data[count] = buffer;
		count++;
	}
	
	int min = data[0];
	int max = data[1];
	int ans = data[2];
	int input;
	
	cout << "Please guess an integer number between " << min << " and " << max << endl; 
	cin >> input;
	while(input != ans){
		if(input < min || input > max){
			cout << "The number you entered is not within the valid range." << endl;
			cout << "Please try again!" << endl;
		}
		else if(input < ans){
			min = input;
		}
		else if(input > ans){
			max = input;
		}
		cout << "Please guess an integer number between " << min << " and " << max << endl; 
		cin >> input;
	}
	cout << "Bingo! You hit the bomb." << endl; 

	system("pause");
	return 0;
}

>>>>>>> 3af4f55 (commit2021/05/07)

File: hw1/java/src/hw1/hw1.java
<<<<<<< HEAD
package hw1;
import java.util.*;
import java.io.*;

public class hw1 {

	public static void main(String[] args) throws IOException {
		FileReader inputFile = new FileReader("input.txt");
		Scanner in = new Scanner(inputFile);
		Scanner input = new Scanner(System.in);
		
		double buffer;	
		
		int[] data = new int[3];
		
		for(int i=0; i<data.length; i++)
		{
			buffer = in.nextDouble();
			
			data[i] = (int)buffer;
			
			//System.out.println(data[i]);
		}
		in.close();
		
		int min = data[0];
		int max = data[1];
		int ans = data[2];
		int cin;
		
		System.out.println("Please guess an integer number between "+ min + " and " + max);
		cin = input.nextInt();
		
		while(cin != ans) 
		{
			if(cin < min || cin > max){
				System.out.println("The number you entered is not within the valid range.");
				System.out.println("Please try again!");
			}
			else if(cin < ans){
				min = cin;
			}
			else if(cin > ans){
				max = cin;
			}
			System.out.println("Please guess an integer number between "+ min + " and " + max);
			cin = input.nextInt();
		}
		System.out.println("Bingo! You hit the bomb.");
	}
}
=======
package hw1;
import java.util.*;
import java.io.*;

public class hw1 {

	public static void main(String[] args) throws IOException {
		FileReader inputFile = new FileReader("input.txt");
		Scanner in = new Scanner(inputFile);
		Scanner input = new Scanner(System.in);
		
		double buffer;	
		
		int[] data = new int[3];
		
		for(int i=0; i<data.length; i++)
		{
			buffer = in.nextDouble();
			
			data[i] = (int)buffer;
			
			//System.out.println(data[i]);
		}
		in.close();
		
		int min = data[0];
		int max = data[1];
		int ans = data[2];
		int cin;
		
		System.out.println("Please guess an integer number between "+ min + " and " + max);
		cin = input.nextInt();
		
		while(cin != ans) 
		{
			if(cin < min || cin > max){
				System.out.println("The number you entered is not within the valid range.");
				System.out.println("Please try again!");
			}
			else if(cin < ans){
				min = cin;
			}
			else if(cin > ans){
				max = cin;
			}
			System.out.println("Please guess an integer number between "+ min + " and " + max);
			cin = input.nextInt();
		}
		System.out.println("Bingo! You hit the bomb.");
	}
}
>>>>>>> 3af4f55 (commit2021/05/07)
